manifests/o/ottercorp/FFXIVQuickLauncherCN/6.2.45-beta2/ottercorp.FFXIVQuickLauncherCN.installer.yaml

```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ottercorp.FFXIVQuickLauncherCN
+PackageVersion: 6.2.45-beta2
+Installers:
+- Architecture: x86
+  InstallerType: exe
+  InstallerUrl: https://github.com/AinaSnow/FFXIVQuickLauncher/releases/download/6.2.45-beta2/Setup.exe

```

Comment:
``
  InstallerUrl: https://github.com/AinaSnow/FFXIVQuickLauncher/releases/download/6.2.45-beta2/Setup.exe
```
Maybe the release should be published to https://github.com/ottercorp/FFXIVQuickLauncher?

Reply:
For some internal reasons we will not release releases in ottercorp/FFXIVQuickLauncher 
I'll come over and update the winget source when we're ready at ottercorp/FFXIVQuickLauncher

Reply:
> I'll come over and update the winget source when we're ready at ottercorp/FFXIVQuickLauncher

It looks like we'll have to wait then since we require the installer to come from https://github.com/ottercorp/FFXIVQuickLauncher instead from a forked repository.